stk/calculators/energy/energy_calculators.py

```diff
@@ -377,3 +382,847 @@ def energy(self, mol, conformer=-1):
         rdkit.GetSSSR(mol.mol)
         ff = rdkit.UFFGetMoleculeForceField(mol.mol, confId=conformer)
         return ff.CalcEnergy()
+
+
+class XTBEnergy(EnergyCalculator):
+    """
+    Uses GFN-xTB to calculate energies and other properties of molecules.
+
+    Notes
+    -----
+    When running :meth:`energy`, this calculator changes the
+    present working directory with :func:`os.chdir`. The original
+    working directory will be restored even if an error is raised so
+    unless multi-threading is being used this implementation detail
+    should not matter.
+
+    If multi-threading is being used an error could occur if two
+    different threads need to know about the current working directory
+    as this :class:`.EnergyCalculator` can change it from under them.
+
+    Note that this does not have any impact on multi-processing,
+    which should always be safe.
+
+    Documentation for xTB available:
+    https://xtb-docs.readthedocs.io/en/latest/setup.html
+
+    Attributes
+    ----------
+    xtb_path : :class:`str`
+        The path to the xTB executable.
+
+    gfn_version : :class:`str`
+        Parameterization of GFN to use in xTB.
+        For details:
+            https://xtb-docs.readthedocs.io/en/latest/basics.html
+
+    output_dir : :class:`str`
+        The name of the directory into which files generated during
+        the optimization are written, if ``None`` then
+        :func:`uuid.uuid4` is used.
+
+    num_cores : :class:`int`
+        The number of cores for xTB to use. Requires appropriate setup
+        of xTB by user.
+
+    etemp : :class:`int`
+        Electronic temperature to use (in K). Defaults to 300K.
+
+    solvent : :class:`str`
+        Solvent to use in GBSA implicit solvation method.
+        For details:
+            https://xtb-docs.readthedocs.io/en/latest/gbsa.html
+
+    solvent_grid : :class:`str`
+        Grid level to use in SASA calculations for GBSA implicit solvent.
+        Options:
+            normal, tight, verytight, extreme
+        For details:
+            https://xtb-docs.readthedocs.io/en/latest/gbsa.html
+
+    charge : :class:`int`
+        Formal molecular charge.
+
+    unpaired_electrons : :class:`int`
+        Number of unpaired electrons.
+
+    use_cache : :class:`bool`
+        If ``True`` :meth:`energy` will not run twice on the same
+        molecule and conformer.
+
+    mem_ulimit : :class: `bool`
+        If ``True`` :meth:`energy` will be run without constraints on
+        the stacksize. If memory issues are encountered, this should be
+        ``True``, however this may raise issues on clusters.
+
+    total_energies : :class:`dict`
+        :class:`dict` of the total energy of each `mol`, `conformer` pair.
+        Key is `(mol, conformer)`
+
+    homo_lumo_gaps : :class:`dict`
+        :class:`dict` of the HOMO-LUMO gap of each `mol`, `conformer` pair.
+        Key is `(mol, conformer)`
```

Comment:
Needs two backticks on each side of the tuple.

Reply:
Needs a full stop at the end of sentence.

Reply:
Rather than saying of each mol conformer pair (which raises the question of what mol and what conformer you are talking about), Say something like, for each 
```
:class:`.Molecule` and conformer passed to :meth:`energy`.

```